.gitignore

```diff
@@ -2,3 +2,5 @@
 .idea
 .vscode
 testing_*.coverprofile
+# Development container configuration (https://containers.dev/)
```

Comment:
unrelated change, maybe via separate commit

Reply:
I can remove it, no worries :slightly_smiling_face: